.github/workflows/ci.yaml

```diff
@@ -34,35 +34,25 @@ jobs:
       - run: make dev
       - run: make test
 
+  setup-integration-test:
+    runs-on: ubuntu-latest
+    outputs:
+      versions: ${{ steps.versions.outputs.versions }}
+    steps:
+        - uses: actions/checkout@v4
+        - id: versions
+          working-directory: ./integration
+          run: |
+            versions=$(make print-versions | jq -c -Rs 'split("\n") | .[:-1]')
```

Comment:
There's a little bit of `jq` magic here. The command is, "output compact, raw input, slurp, split on new lines, and remove the last element"